core/services/vrf/v2/listener_v2.go

```diff
@@ -1385,7 +1405,7 @@ func (lsn *listenerV2) simulateFulfillment(
 					// that's all we need in the event of a force-fulfillment.
 					m := trr.Result.Value.(map[string]any)
 					res.payload = m["output"].(string)
-					res.proof = m["proof"].(vrf_coordinator_v2.VRFProof)
+					res.proof = FromV2Proof(m["proof"].(vrf_coordinator_v2.VRFProof))
 					res.reqCommitment = NewRequestCommitment(m["requestCommitment"])
```

Comment:
Do we need to add a condition to this enclosing logic to see if the task type is TaskTypeVRFV2Plus?

Reply:
I think it's ok because in the above `if`, we check if pipeline task is `VRFV2` instead of `VRFV2Plus`

Reply:
Right, what I'm saying is that in the above if we _only_ check for VRFV2, and not VRFV2Plus, so we are not collecting the "txData for the fulfillRandomWords call" for VRFV2Plus. This seems like it has the potential to be an edge-case for us. Maybe need to add an if for VRFV2Plus

Reply:
do we ever need the txData? this logic is only for force fulfilling and we never force fulfill for VRFV2Plus. If we ever try to force fulfill for VRFV2Plus due to a software bug, it's better to have empty res so we fail the fulfillment and find out about it?

Reply:
do we ever need the txData? this logic is only for force fulfilling and we never force fulfill for VRFV2Plus. If we ever try to force fulfill for VRFV2Plus due to a software bug, it's better to have empty res so we fail the fulfillment and get alarmed about it?